### PR TITLE
Update overview.md

### PR DESCRIPTION
**Pull request by wuzhong-papermoon**
Is "ContanerChain" a typo here?

### Description

Please explain the changes this PR addresses here.

### Checklist

- [ ] I have added a label to this PR 🏷️
- [✔ ] I have run my changes through Grammarly
- [✔ ] If this page requires a disclaimer, I have added one
- [✔ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

**Comment by albertov19** [comment]
@wuzhong-papermoon not really - We used to name Tanssi Appchains "ContainerChains" but we've moved away from that terminology in favor of appchain, or Tanssi appchains.

Still we need to modify it here to the new standards

File: builders/build/templates/overview.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ More information about how to configure the Cumulus SDK to integrate a Tanssi ap
 
 ## Base Setup to Support the Tanssi Protocol {: #base-setup-supporting-tanssi }
 
-Besides Cumulus, a ContanerChain must implement the following Tanssi modules to support the protocol and benefit safely from Tanssi's block production as a service:
+Besides Cumulus, a Container Chain must implement the following Tanssi modules to support the protocol and benefit safely from Tanssi's block production as a service:
```
Comment (eshaben) [inline builders/build/templates/overview.md:37]:
```suggestion
Besides Cumulus, an appchain must implement the following Tanssi modules to support the protocol and benefit safely from Tanssi's block production as a service:
```

**Review by eshaben — COMMENTED** [review]
Yeah, as Alberto mentioned, we moved away from "ContainerChains" this one was probably missed because it had a typo in it 🙂

**Review by eshaben — APPROVED** [review]
LGTM!

**Pull request merged**